packages/runtime/__tests__/expression.test.ts

```diff
@@ -0,0 +1,34 @@
+import { parseExpression } from '../src/store';
+
+describe('parseExpression function', () => {
+  it('can parse {{}} expression', () => {
+    expect(parseExpression('{{ value }}')).toMatchObject([
+      { expression: 'value', isDynamic: true },
+    ]);
+    expect(parseExpression('Hello, {{ value }}!')).toMatchObject([
+      { expression: 'Hello, ', isDynamic: false },
+      { expression: 'value', isDynamic: true },
```

Comment:
the result has drop the last `!` character, is this expected?

Reply:
It an expected bug. I fixed it.